### ISSUE TITLE
Color contrast checker fails on hex with alpha

### ISSUE THREAD

**Issue by ENTL210** [HundredVisionsGuy/webcode-tk#18]
Also with rgba and I assume hsla colors.

See what we can do.

**Comment by HundredVisionsGuy** [comment]
We solved it with commit: 804d79d5e17db83d96edcae42409be2a481a62e2
